test/refactoring/DocCommentBoilerplate/Outputs/with_parameters/with_parameters.swift.expected

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Make sure to update the tests

Reply:
Just updated them. Running the test suite now. Will push when done. In the mean time will figure out how to handle return and throws.

Do we also want this feature for constructors?

Reply:
Yep, and also subscripts.

Reply:
Great! Really excited to have come this far!